### PR TITLE
Dev install acorn, espree and mkdirp modules

### PR DESCRIPTION
**Pull request by archmoj**
In respect to #https://github.com/plotly/plotly.js/issues/4796
This PR helps reduce the number of warnings for `acorn`using `regl`.

Before: 
before

After:
after

See commit https://github.com/plotly/plotly.js/commit/cabe146d31410f2a6544df84cff9b80f746b7f61 for details.

Also to mention, the audit files were created using  the commands below:
```
npm audit --json > audit.json
npm audit | sed -r "s/\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]//g" > audit.txt
```

@plotly/plotly_js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "@turf/area": "^6.0.1",
     "@turf/bbox": "^6.0.1",
     "@turf/centroid": "^6.0.2",
+    "acorn": "^7.1.1",
```
Comment (alexcjohnson) [inline package.json:64]: Is it necessary that this lives in `dependencies`, rather than `devDependencies`?

Comment (archmoj) [reply package.json:64]: Good catch!

Comment (archmoj) [reply package.json]: Moved in https://github.com/plotly/plotly.js/pull/4800/commits/8a96b50696ff6fe286f5e5d626d2de4164addc88.

**Comment by archmoj** [comment]
Also dev-installed `espree` v6.2.1 and `mkdirp` v1.0.4 in this PR to reduce the warnings to:

newAfter

**Pull request merged**